src/modules/Lan.rb

```diff
@@ -1087,11 +1086,15 @@ def configure_as_bridge!(ifcfg, bridge_name)
     end
 
     # Convenience method that returns true if the current item has link and can
-    # be enslabed in a bridge.
+    # be enslaved in a bridge.
     #
     # @return [Boolean] true if it is bridgeable
     def connected_and_bridgeable?(bridge_name, item, config)
-      if !LanItems.IsBridgeable(bridge_name, item)
+      # FIXME: a workaround until we fully use builders in proposals
+      bridge_builder = Y2Network::InterfaceConfigBuilder.for("br")
```

Comment:
Using `Y2Network::InterfaceType` would be desirable :)

Reply:
well InterfaceConfigBuilder hasn't use InterfaceType up to now ... changing it deserve own PR in my POV as I haven't implement the ```.for``` functionality here